src/unstract/sdk/adapters/vectordb/exceptions.py

```diff
@@ -0,0 +1,37 @@
+from qdrant_client.http.exceptions import ApiException as QdrantAPIException
+
+from unstract.sdk.adapters.vectordb.qdrant.src import Qdrant
+from unstract.sdk.adapters.vectordb.vectordb_adapter import VectorDBAdapter
+from unstract.sdk.exceptions import VectorDBError
```

Comment:
Is VectorDBError defined in exceptions? I see that it is removed in this PR(?)

Reply:
Yes, its always been present and it inherits from `SdkError`